File: Bridge/System/IO/MemoryStream.cs
using Bridge;

namespace System.IO
{
    public class MemoryStream : Stream
    {       
        public MemoryStream() : this(0)
        {
        }

        public MemoryStream(int capacity)
        {
        }

        public MemoryStream(byte[] buffer)
        {
        }

        public MemoryStream(byte[] buffer, bool writable)
        {
        }

        public MemoryStream(byte[] buffer, int index, int count)
        {
        }

        public MemoryStream(byte[] buffer, int index, int count, bool writable)
        {
        }

        public MemoryStream(byte[] buffer, int index, int count, bool writable, bool publiclyVisible)
        {
        }
        
        public override bool CanRead
        {
            get { throw new NotImplementedException(); }
        }

        public override bool CanSeek
        {
            get { throw new NotImplementedException(); }
        }

        public override bool CanWrite
        {
            get { throw new NotImplementedException(); }
        }

        public virtual int Capacity
        {
            get
            {
                throw new NotImplementedException();
            }

            set
            {
                throw new NotImplementedException();
            }
        }

        public override long Length
        {
            get
            {
                throw new NotImplementedException();
            }
        }

        public override long Position
        {
            get
            {
                throw new NotImplementedException();
            }

            set
            {
                throw new NotImplementedException();
            }
        }

        protected override void Dispose(bool disposing)
        {
            throw new NotImplementedException();
        }

        public override void Flush()
        {
            throw new NotImplementedException();
        }

        public virtual byte[] GetBuffer()
        {
            throw new NotImplementedException();
        }

        public byte[] GetByteArray()
        {
            throw new NotImplementedException();
        }

        public override int Read(byte[] buffer, int offset, int count)
        {
            throw new NotImplementedException();
        }

        public override int ReadByte()
        {
            throw new NotImplementedException();
        }

        public override long Seek(long offset, SeekOrigin loc)
        {
            throw new NotImplementedException();
        }
        
        public override void SetLength(long value)
        {
            throw new NotImplementedException();
        }

        public virtual byte[] ToArray()
        {
            throw new NotImplementedException();
        }

        public override void Write(byte[] buffer, int offset, int count)
        {
            throw new NotImplementedException();
        }

        public override void WriteByte(byte value)
        {
            throw new NotImplementedException();
        }

        public virtual void WriteTo(Stream stream)
        {
            throw new NotImplementedException();
        }
    }
}

File: Bridge/Resources/.generated/system/iO/memoryStream.js
    Bridge.define("System.IO.MemoryStream", {
        inherits: [System.IO.Stream],
        props: {
            CanRead: {
                get: function () {
                    throw new System.NotImplementedException();
                }
            },
            CanSeek: {
                get: function () {
                    throw new System.NotImplementedException();
                }
            },
            CanWrite: {
                get: function () {
                    throw new System.NotImplementedException();
                }
            },
            Capacity: {
                get: function () {
                    throw new System.NotImplementedException();
                },
                set: function (value) {
                    throw new System.NotImplementedException();
                }
            },
            Length: {
                get: function () {
                    throw new System.NotImplementedException();
                }
            },
            Position: {
                get: function () {
                    throw new System.NotImplementedException();
                },
                set: function (value) {
                    throw new System.NotImplementedException();
                }
            }
        },
        ctors: {
            ctor: function () {
                System.IO.MemoryStream.$ctor6.call(this, 0);
            },
            $ctor6: function (capacity) {
                this.$initialize();
                System.IO.Stream.ctor.call(this);
            },
            $ctor1: function (buffer) {
                this.$initialize();
                System.IO.Stream.ctor.call(this);
            },
            $ctor2: function (buffer, writable) {
                this.$initialize();
                System.IO.Stream.ctor.call(this);
            },
            $ctor3: function (buffer, index, count) {
                this.$initialize();
                System.IO.Stream.ctor.call(this);
            },
            $ctor4: function (buffer, index, count, writable) {
                this.$initialize();
                System.IO.Stream.ctor.call(this);
            },
            $ctor5: function (buffer, index, count, writable, publiclyVisible) {
                this.$initialize();
                System.IO.Stream.ctor.call(this);
            }
        },
        methods: {
            dispose$1: function (disposing) {
                throw new System.NotImplementedException();
            },
            flush: function () {
                throw new System.NotImplementedException();
            },
            getBuffer: function () {
                throw new System.NotImplementedException();
            },
            getByteArray: function () {
                throw new System.NotImplementedException();
            },
            read: function (buffer, offset, count) {
                throw new System.NotImplementedException();
            },
            readByte: function () {
                throw new System.NotImplementedException();
            },
            seek: function (offset, loc) {
                throw new System.NotImplementedException();
            },
            setLength: function (value) {
                throw new System.NotImplementedException();
            },
            toArray: function () {
                throw new System.NotImplementedException();
            },
            write: function (buffer, offset, count) {
                throw new System.NotImplementedException();
            },
            writeByte: function (value) {
                throw new System.NotImplementedException();
            },
            writeTo: function (stream) {
                throw new System.NotImplementedException();
            }
        }
    });
